tests/emails/user_notification_after_stock_update_test.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+from unittest.mock import patch
+
+from emails.user_notification_after_stock_update import retrieve_data_to_warn_user_after_stock_update_affecting_booking
+from tests.model_creators.generic_creators import create_user, create_offerer, create_venue, create_booking
+from tests.model_creators.specific_creators import create_offer_with_event_product, create_event_occurrence, \
+    create_stock_from_event_occurrence
+
+
+class RetrieveDataToWarnUserAfterStockUpdateAffectingBookingTest:
+    @patch('emails.user_notification_after_stock_update.feature_send_mail_to_users_enabled', return_value=True)
```

Comment:
```suggestion
    @patch('emails.user_notification_after_stock_update.feature_send_mail_to_users_enabled')
```
Et mettre le `return_value` dans le `given`.
Je te laisse régler tous les autres.

Reply:
Ok, je ne connaissais pas cette convention